common/source/docs/common-ark-fpv.rst

```diff
@@ -0,0 +1,394 @@
+-------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Signal Name
+     - Voltage
+   * - 1
+     - 5.0V
+     - 5.0V
+   * - 2
+     - SPI6_SCK
+     - 3.3V
+   * - 3
+     - SPI6_MISO
+     - 3.3V
+   * - 4
+     - SPI6_MOSI
+     - 3.3V
+   * - 5
+     - SPI6_nCS1
+     - 3.3V
+   * - 6
+     - SPI6_DRDY1
+     - 3.3V
+   * - 7
+     - SPI6_nRESET
+     - 3.3V
+   * - 8
+     - GND
+     - GND
+
+
+Flight Controller Debug - 6 Pin JST-SH
+--------------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Signal Name
+     - Voltage
+   * - 1
+     - 3V3_FMU
+     - 3.3V
+   * - 2
+     - USART4_TX_DEBUG
+     - 3.3V
+   * - 3
+     - USART4_RX_DEBUG
+     - 3.3V
+   * - 4
+     - FMU_SWDIO
+     - 3.3V
+   * - 5
+     - FMU_SWCLK
+     - 3.3V
+   * - 6
+     - GND
+     - GND
+
+
+UART Mapping
+============
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Function
+   * - SERIAL0
+     - USB
+   * - SERIAL1
+     - UART7 (Telem)
+   * - SERIAL2
+     - UART5 (DisplayPort HD VTX)
+   * - SERIAL3
+     - USART1 (GPS1)
+   * - SERIAL4
+     - USART2 (User, SBUS pin on HD VTX, RX only)
+   * - SERIAL5
+     - UART4 (ESC Telem, RX only)
+   * - SERIAL6
+     - USART6 (RC Input)
+   * - SERIAL7
+     - OTG2 (SLCAN)
+
+
+All UARTS support DMA. Any UART may be re-tasked by changing its protocol parameter.
+
+RC Input
+========
+RC input is configured on the RX6 (UART6_RX) pin. It supports all RC protocols except PPM. See :ref:`Radio Control Systems <common-rc-systems>` for details for a specific RC system. :ref:`SERIAL6_PROTOCOL<SERIAL6_PROTOCOL>` is set to “23”, by default, to enable this.
+
+
+* SBUS/DSM/SRXL connects to the RX6 pin.
+* FPort requires connection to TX6 and :ref:`SERIAL6_OPTIONS<SERIAL2_OPTIONS>` be set to “7”.
+* CRSF also requires a TX6 connection, in addition to RX6, and automatically provides telemetry. Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>`
+* SRXL2 requires a connecton to TX6 and automatically provides telemetry. Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` to “4”.
+
+Battery Monitoring
+==================
+The board has a internal voltage sensor and connections on the ESC connector for an external current sensor input. The board supports up to 12S LiPo batteries.
+
+The default battery parameters are:
+
+* :ref:`BATT_MONITOR<BATT_MONITOR>` = 4
+* :ref:`BATT_VOLT_PIN<BATT_VOLT_PIN__AP_BattMonitor_Analog>` = 9
+* :ref:`BATT_CURR_PIN<BATT_CURR_PIN__AP_BattMonitor_Analog>` = 12
+* :ref:`BATT_VOLT_MULT<BATT_VOLT_MULT__AP_BattMonitor_Analog>` = 21
+* :ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT__AP_BattMonitor_Analog>` = 120
+
+Compass
+=======
+This autopilot has a built-in IIS2MDC compass.Due to potential interference, the autopilot is usually used with an external I2C compass as part of a GPS/Compass combination.
+
+OSD Support
+===========
+This flight controller has an MSP-DisplayPort output on a 6-pin DJI-compatible JST SH.
```

Comment:
Line 238 states is not standard pinout, should it also be noted here ?

Reply:
cant hurt...